packages/sdk-metrics/src/export/MetricData.ts

```diff
@@ -14,18 +14,30 @@
  * limitations under the License.
  */
 
-import { HrTime, MetricAttributes } from '@opentelemetry/api';
+import { HrTime, MetricAttributes, ValueType } from '@opentelemetry/api';
 import { InstrumentationScope } from '@opentelemetry/core';
 import { IResource } from '@opentelemetry/resources';
-import { InstrumentDescriptor } from '../InstrumentDescriptor';
+import { InstrumentType } from '../InstrumentDescriptor';
 import { AggregationTemporality } from './AggregationTemporality';
 import { Histogram, ExponentialHistogram } from '../aggregator/types';
 
+export interface MetricDescriptor {
```

Comment:
Well, it seems we have to keep the public name `InstrumentDescriptor` unchanged to avoid breakage. I've mapped the new `MetricDescriptor` (which is identical to the old `InstrumentDescriptor`) to the publicly exported name `InstrumentDescriptor`.